### PR TITLE
Update disable-dpi-awareness.md

### PR DESCRIPTION
**Pull request by AlexB-SheldonMFG**
#5664, Write-the-Docs-2020

### CODE REVIEW

**Comment by PRMerger16** [comment]
@AlexB-SheldonMFG : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/designers/disable-dpi-awareness.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ When Visual Studio runs as a DPI-unaware process, the designer layout issues are
...
 
-It's important to restart Visual Studio as a DPI-aware process when you're finished working in the **Windows Forms Designer**. When it's running as a DPI-unaware process, fonts can look blurry and you may see issues in other designers such as the **XAML Designer**. If you close and reopen Visual Studio when it's running in DPI-unaware mode, it becomes DPI-aware again. You can also click the **Restart Visual Studio as a DPI-aware process** option in the informational bar.
+It's important to restart Visual Studio as a DPI-aware process when you're finished working in the **Windows Forms Designer**. When it's running as a DPI-unaware process, fonts can look blurry and you may see issues in other designers such as the **XAML Designer**. If you close and reopen Visual Studio when it's running in DPI-unaware mode, it becomes DPI-aware again. You can also slect the **Restart Visual Studio as a DPI-aware process** option in the informational bar.
```
Comment (gregvanl) [inline docs/designers/disable-dpi-awareness.md:51]: typo "slect" should be "select"

**Comment by PRMerger8** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@gregvanl I've applied the spelling fix. I'll merge the PR now. Thanks.

**Pull request merged**